sys/linux/bpf_amd64.const

```diff
@@ -229,7 +229,7 @@ BTF_KIND_VOLATILE = 9
 BTF_MAGIC = 60319
 BTF_VERSION = 1
 MAX_BPF_REG = 11
-__BPF_FUNC_MAX_ID = 141
+__BPF_FUNC_MAX_ID = 136
```

Comment:
These unrelated changes are confusing.  It would be preferable to either (a) use the same linux-next version that the constants were generated from most recently, so that there are no unnecessary changes, or (b) upgrade to the latest linux-next version (e.g. currently next-20200706) and mention this in the commit message.

Reply:
Done, thanks!